lib/tasks/migration.rake

```diff
@@ -74,15 +74,17 @@ namespace :migration do
   end
 
   desc "batch migrate generic files from modified ERA FOXML file"
-  task :eraitem, [:dir, :migrate_datastreams] => :environment do |t, args|
+  task :eraitem, [:dir, :migrate_datastreams, :usecomm, :usecoll] => :environment do |t, args|
     args.with_defaults(:migrate_datastreams => "true")
     begin
       MigrationLogger.info "**************START: Migrate ERA objects *******************"
       metadata_dir = args.dir
       migrate_datastreams = args.migrate_datastreams == "true"
-      # Usage: Rake migration:eraitem[<file directory here, path included>,<optional: migrate_datastreams: boolean>]
+      usecomm = args.usecomm
+      usecoll = args.usecoll
```

Comment:
You could just pass the args into the method below instead of assigning it to a variable? Considering nothing is using this variable again.